### DISCUSSION TITLE
I can't find the local database

### DISCUSSION THREAD

**Discussion by VlaK0r** [usememos/memos#1944] [Q&A]
Hi!
I can't find the local database on disk
Where is it located?

**Comment by boojack** [comment]
Could you provide the startup command?

**Reply by VlaK0r** [reply]
docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos ghcr.io/usememos/memos:latest
On the VPS server I found.
I can't find it on the local PC

**Reply by boojack** [reply]
The data will only be stored on your VPS.

**Reply by VlaK0r** [reply]
Docker is installed on the local PC. Memos is installed with it.
Тhere won't be a database for this instance anyway?

**Reply by VlaK0r** [reply]
I have 2 versions installed, on the server and on the PC.
I found the database on the server, but not on the PC
